lightning/mydump/parquet_parser_test.go

```diff
@@ -81,3 +79,69 @@ func (s testParquetParserSuite) TestParquetParser(c *C) {
 
 	c.Assert(reader.ReadRow(), Equals, io.EOF)
 }
+
+func (s testParquetParserSuite) TestParquetVariousTypes(c *C) {
+	type Test struct {
+		Date            int32 `parquet:"name=date, type=DATE"`
+		TimeMillis      int32 `parquet:"name=timemillis, type=TIME_MILLIS"`
+		TimeMicros      int64 `parquet:"name=timemicros, type=TIME_MICROS"`
+		TimestampMillis int64 `parquet:"name=timestampmillis, type=TIMESTAMP_MILLIS"`
+		TimestampMicros int64 `parquet:"name=timestampmicros, type=TIMESTAMP_MICROS"`
+
+		Decimal1 int32  `parquet:"name=decimal1, type=DECIMAL, scale=2, precision=9, basetype=INT32"`
+		Decimal2 int32  `parquet:"name=decimal2, type=DECIMAL, scale=4, precision=4, basetype=INT32"`
+		Decimal3 int64  `parquet:"name=decimal3, type=DECIMAL, scale=2, precision=18, basetype=INT64"`
+		Decimal4 string `parquet:"name=decimal4, type=DECIMAL, scale=2, precision=10, basetype=FIXED_LEN_BYTE_ARRAY, length=12"`
+		Decimal5 string `parquet:"name=decimal5, type=DECIMAL, scale=2, precision=20, basetype=BYTE_ARRAY"`
+	}
+
+	dir := c.MkDir()
+	// prepare data
+	name := "test123.parquet"
+	testPath := filepath.Join(dir, name)
+	pf, err := local.NewLocalFileWriter(testPath)
+	c.Assert(err, IsNil)
+	test := &Test{}
+	writer, err := writer2.NewParquetWriter(pf, test, 2)
+	c.Assert(err, IsNil)
+
+	v := &Test{
+		Date:            18564,              //2020-10-29
+		TimeMillis:      62775123,           // 17:26:15.123
+		TimeMicros:      62775123000,        // 17:26:15.123
+		TimestampMillis: 1603963672356,      // 2020-10-29T17:27:52.356
+		TimestampMicros: 1603963672356956,   //2020-10-29T17:27:52.356956
+		Decimal1:        -12345678,          // -123456.78
+		Decimal2:        456,                // 0.0456
+		Decimal3:        123456789012345678, //1234567890123456.78
```

Comment:
add a test case for these decimals:

* `0.000`
* `1.000`
* `-1.000`
* `0.999`
* `-0.999`
* `-0.0001`